.github/workflows/auto-update-tools.yml

```diff
@@ -0,0 +1,55 @@
+name: "Automation: Update tooling versions"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
`m`: I think it makes sense to explain directly here why we can't use the dependabot.